build/build-binaries.sh

```diff
@@ -0,0 +1,26 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#!/usr/bin/env bash
+
+# Helper to build release artifacts - binaries only.
+#
+# Usage (from repo root):
+# ./build/build-binaries.sh -o . -v v0.1.0
+
+# Collects args and defines build functions.
+source "$(dirname "${0}")/build-include.sh"
+
+# Build binaries only.
+build_binaries
```

Comment:
Can we use something like a make file and user can just say what they want to build?

e.g.

make binaries v0.1.0
make templates v.02.0 (append templates- automatically)

Reply:
From what I can tell, you're not really supposed to be calling make with arguments, only targets.

Reply:
I'll see if there's a sensible way to do it with environment variables or something.

Reply:
No wories then, this is fine